src/sage/modular/modsym/p1list_nf.py

```diff
@@ -127,15 +128,15 @@ class MSymbol(SageObject):
       level N.
 
     - ``d`` -- (optional) when present, it must be an integral element such
-      that <c> + <d> + N = R, where R is the corresponding ring of integers.
+      that `\la c\ra + \la d\ra + N = R, where `R` is the corresponding ring of integers.
```

Comment:
Not sure if there are pre-defined shortcuts for left and right angles, but if there are they don't seem to be `\la` and `\ra`
```suggestion
      that `\langle c\rangle + \langle d\rangle + N = R`, where `R` is the corresponding ring of integers.
```